### PR TITLE
[iOS] Apply floor map image

### LINKED ISSUE #690: [iOS] [Needed by today 9/23] Apply floor map image
**Idea Description**

- iOS version for #688 

**Reference images and links**

### PR DESCRIPTION
**Pull request by betat**
## Issue
- close #690 

## Overview (Required)
- Replace temporary image with floor map image

## Links
- [Figma](https://www.figma.com/file/NcSMs6dMsD88d4wOY0g3rK/DroidKaigi-2022-Conference-App?node-id=2354%3A3623)

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: app-ios/Sources/MapFeature/MapView.swift [hunk 1]
```diff
@@ -39,9 +36,10 @@ public struct MapView: View {
         WithViewStore(store) { viewStore in
             NavigationView {
                 VStack(alignment: .center) {
```
Comment (ry-itto) [inline app-ios/Sources/MapFeature/MapView.swift:38]: `VStack` already exists, but is this necessary?
If you don't mind, can you remove that and align image center? 🙏🏼

**Review by ry-itto — APPROVED** [review]
Looks good! Thank you for fast implementation!

File: app-ios/Sources/MapFeature/MapView.swift (hunk 1 above)
Comment (betat) [reply app-ios/Sources/MapFeature/MapView.swift]: I fixed in 81a7045e9ffab049650f1461d1ca1cd644d50d4e

**Comment by ry-itto** [comment]
Thanks! I'll merge this 🙏🏼

**Pull request merged**